projects/plugins/jetpack/tests/php/sync/test_class.jetpack-sync-callables.php

```diff
@@ -884,6 +884,10 @@ public function test_plugin_action_links_get_synced() {
 			),
 		);
 
+		if ( ! is_multisite() ) {
```

Comment:
I found out via the tests that My Jetpack doesn't show up on multisites, therefore the link will not either